apps/src/aichat/views/ChatMessage.tsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import styles from './chatMessage.module.scss';
+import classNames from 'classnames';
+
+interface ChatMessageProps {
+  id: string;
+  name: string;
+  role: string;
+  chatMessageText: string;
+  status: string;
+}
+
+const INAPPROPRIATE_MESSAGE = 'This message has been flagged as inappropriate.';
+const TOO_PERSONAL_MESSAGE = 'This message has been flagged as too personal.';
+
+const isAssistant = (role: string) => {
+  return role === 'assistant';
+};
+
+const isUser = (role: string) => {
+  return role === 'user';
+};
+
+const displayUserMessage = (status: string, chatMessageText: string) => {
+  if (status === 'ok') {
+    return (
+      <div
+        id={'chat-workspace-message-body'}
```

Comment:
These IDs are handy for debugging, but for elements that are used more than once, it should be a class name rather than an ID, because each ID should only appear in the DOM once.

Reply:
Thanks! I'll go ahead and remove the `id` since I have the classNames included by type of message (user, inappropriate, or tooPersonal).